src/utils/artifact/generateAragonArtifact.ts

```diff
@@ -46,45 +48,40 @@ function _generateAragonArtifact(
     abi,
     // Additional metadata
     flattenedCode: './code.sol',
-    appName: appEnsName,
-    appId: getAppId(appEnsName)
+    appName,
+    appId: getAppId(appName)
   }
 }
 
 // Function Overloading logic
 
 /**
  * Returns aragon artifact.json from app data
- * @param appEnsName "finance.aragonpm.eth"
  * @param arapp
  * @param abi
  * @param functions Parsed contract function info
  */
 export function generateAragonArtifact(
-  appEnsName: string,
   arapp: AragonAppJson,
   abi: AbiItem[],
   functions: AragonContractFunction[]
 ): AragonArtifact
 
 /**
  * Returns aragon artifact.json from app data
- * @param appEnsName "finance.aragonpm.eth"
  * @param arapp
  * @param abi
  * @param flatCode Flat code of target contract plus all imports
  * @param contractName Target contract name or path: "Finance" | "contracts/Finance.sol"
  */
 export function generateAragonArtifact(
-  appEnsName: string,
   arapp: AragonAppJson,
   abi: AbiItem[],
   flatCode: string,
```

Comment:
As Brett mentioned in other PR, maybe we can use `sourceCode` here to be more consistent?

Reply:
Can we do that in another PR?